docs/integrations/dydx.mdx

```diff
@@ -137,37 +136,37 @@ for API keys to fill in below.
 }
 ```
 
-Once you've created this file, you may run your sidecar as follows:
+Once you've created this file, you may run your connect sidecar as follows:
 
 ```
 slinky --marketmap-provider dydx_api --market-map-endpoint https://<dydx-node REST API> --oracle-config <path to oracle.json>
 ```
 
 ### **Q: How do I know if my validator is properly fetching prices and posting them to the chain?**
 
-**A:** A full set of prometheus metrics are integrated into both the sidecar and the dYdX application binary.
+**A:** A full set of prometheus metrics are integrated into both connect and the dYdX application binary.
 
-A comprehensive overview of the relevant side-car metrics is hosted in the root [metrics.md](https://github.com/skip-mev/slinky/blob/main/metrics.md). This document includes a grafana dashboard that can be utilized to monitor the sidecar. Apart from the Slinky SideCar Dashboard, we highly recommend the [LavenderFive](https://github.com/LavenderFive/slinky-monitoring) and [RhinoStake](https://github.com/RhinoStake/slinky_monitoring_dashboard) monitoring dashboards which are built to be used with the Slinky sidecar.
+A comprehensive overview of the relevant connect metrics is hosted in the root [metrics.md](https://github.com/skip-mev/slinky/blob/main/metrics.md). This document includes a grafana dashboard that can be utilized to monitor the sidecar. Apart from the Connect SideCar Dashboard, we highly recommend the [LavenderFive](https://github.com/LavenderFive/slinky-monitoring) and [RhinoStake](https://github.com/RhinoStake/slinky_monitoring_dashboard) monitoring dashboards which are built to be used with Connect.
```

Comment:
this is another case where we mention say "connect sidecar" in line 139, but change it to just "Connect" here